lib/questions.rb

```diff
@@ -155,30 +187,38 @@ def get_domain_name_from_email_address(email)
 # 'the lion the witch and the wardrobe' becomes
 # 'The Lion the Witch and the Wardrobe'
 def titleize_a_string(string)
+  string.split.map{|word| ['and','the'].include?(word) ? word : word.capitalize}.join(" ").sub(/./, &:upcase)
```

Comment:
Line is too long. [109/80]<br>Space missing after comma.<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.